guides/common/modules/proc_excluding-hosts-from-rh-cloud-and-insights-client-reports.adoc

```diff
@@ -4,16 +4,18 @@
 You can set the `host_registration_insights` parameter to *False* to omit _rh-cloud_ and _insights-client_ reports.
 {Project} will exclude the hosts from _rh-cloud_ reports and block _insights-client_ from uploading a report to the cloud.
 
-Use the following procedure to change the value of `host_registration_insights` parameter:
+Use the following procedure to change the value of `host_registration_insights` parameter.
```

Comment:
You can drop this line to avoid using self-referential language ("the following procedure").

Reply:
Removed.